### PR TITLE
OKTA-756271 - Minor telephony hook updates

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added failover RL limit info to telephony hook API and inline hook concepts. Ran acrolinx checks on three topics.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-756271](https://oktainc.atlassian.net/browse/OKTA-756271 )

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/5a2292f5388e899e5c9be638569e416d5c53ca5f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3fc52fe5-22cd-4dc7-a1e9-5d0ce5a19bff) | 86  | 64  | 74  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/5a2292f5388e899e5c9be638569e416d5c53ca5f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/13264da3-87f6-446b-8081-3fecc6a45734) | 90  | 76  | 96  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/telephony-hook/index.md](https://github.com/okta/okta-developer-docs/blob/5a2292f5388e899e5c9be638569e416d5c53ca5f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Ftelephony-hook%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ca625907-fd80-4c5e-9f0c-c9b75cb871ea) | 87  | 62  | 93  | 96  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.278a7298-be85-4e00-9611-f2f037e8904d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/aa486824df72582c1efc43c2b7090537bc0555fd/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/002c9cb8-a923-4a0b-9fc2-bfb346ccf2f6) | 86  | 64  | 74  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/aa486824df72582c1efc43c2b7090537bc0555fd/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/89e93e3d-f25f-4774-b756-a2d8d6cb0fbf) | 90  | 76  | 96  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/telephony-hook/index.md](https://github.com/okta/okta-developer-docs/blob/aa486824df72582c1efc43c2b7090537bc0555fd/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Ftelephony-hook%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3ce38789-2a87-4479-bda2-05d4d11bd238) | 87  | 62  | 93  | 96  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e4c81126-f7ee-411f-a733-c726af1c3bbe?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/8cda9884d93a193bff46a7b1482f088d08c01256/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2505008b-dca5-4ffa-a180-956fdf7a95c7) | 86  | 64  | 74  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/8cda9884d93a193bff46a7b1482f088d08c01256/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/169098aa-24af-47bc-8986-7a413cc5a1b6) | 90  | 76  | 96  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/telephony-hook/index.md](https://github.com/okta/okta-developer-docs/blob/8cda9884d93a193bff46a7b1482f088d08c01256/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Ftelephony-hook%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a55e7f97-0982-4185-863a-2e702cf529c4) | 87  | 62  | 93  | 96  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.a53a8578-173c-4526-a3f1-f24a81a141dd?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This guide provides an example of an Okta telephony inline hook. This guide uses
 
 > **Note:** You can also use Amazon Web Services Lambda Serverless Framework as an external service. See [Setting Up Serverless Framework With AWS](https://www.serverless.com/framework/docs/getting-started).
 
-* Make sure you have a user in your org with a Phone authenticator enrolled. See [MFA Usage report](https://help.okta.com/okta_help.htm?type=oie&id=ext-mfa-usage).
+* Ensure you have a user in your org with a Phone authenticator enrolled. See [MFA Usage report](https://help.okta.com/okta_help.htm?type=oie&id=ext-mfa-usage).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:35]:
```suggestion
* Ensure that you have a user in your org with a Phone authenticator enrolled. See [MFA Usage report](https://help.okta.com/okta_help.htm?type=oie&id=ext-mfa-usage).
```

**Review by susanharper-okta — COMMENTED** [review]
one nit

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/d82dfdd3d0ba2fb5e2a9e113fa95f638890adda0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/33825dd4-26f8-40ad-bff9-d142a4eabaf8) | 86  | 64  | 74  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/d82dfdd3d0ba2fb5e2a9e113fa95f638890adda0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/913a7a4d-08c9-49cc-9c55-345d2f5c951b) | 90  | 76  | 96  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/telephony-hook/index.md](https://github.com/okta/okta-developer-docs/blob/d82dfdd3d0ba2fb5e2a9e113fa95f638890adda0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Ftelephony-hook%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b663e7aa-8881-4586-b1f0-5a4d85e8b855) | 87  | 62  | 93  | 96  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.6dedc9a3-cf37-4cc6-b9ec-7fe7eeb91a98?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Comment by mauricesharp-okta** [comment]
A few misc. comments:
- AFAIK the timeout for Okta fallback is effectively 6 seconds... two failures of 3 seconds. If I understand it correctly, the first timeout causes another call to the inline hook and the second failure triggers the fallback.
- I didn't see a mention of the rate limit for calling the telephony hook. Okta rate limits the call to that hook to manage toll-fraud

**Comment by brianduffield-okta** [comment]
Thanks for your comments, Maurice. I opened another ticket to address them and review another item that affects these docs, see: https://oktainc.atlassian.net/browse/OKTA-793240

**Pull request merged**